docs/user-guide/src/capm3/pivoting.md

```diff
@@ -90,34 +87,34 @@ This can now be achieved with the following procedure:
 
 3. Use clusterctl init to install the provider components
 
-    Example:
+   Example:
 
-    `clusterctl init --infrastructure metal3:v0.3.1
---target-namespace metal3 --watching-namespace metal3`
+   `clusterctl init --infrastructure metal3:v1.1.
+   --target-namespace metal3 --watching-namespace metal3`
 
-    This command will create the necessary CAPI controllers (CAPI, CABPK, CAKCP)
-  and CAPM3 as the infrastructure provider which also includes the Baremetal
-  Operator (BMO). All of the controllers will be installed on namespace `metal3`
-  and they will be watching over objects in namespace `metal3`.
+ This command will create the necessary CAPI controllers (CAPI, CABPK, CAKCP)
+ and CAPM3 as the infrastructure provider which also includes the Baremetal
+ Operator (BMO). All of the controllers will be installed on namespace `metal3`
+ and they will be watching over objects in namespace `metal3`.
 
    In `metal3-dev-env` steps 1 and 2 are part of `make` command.
 4. Provision target cluster:
 
-    Example:
+   Example:
 
-    `clusterctl config cluster ... | kubectl apply -f -`
+   `clusterctl config cluster ... | kubectl apply -f -`
 
-    In `metal3-dev-env` this is done through the scripts in `scripts/v1alphaX/`
-  directory.
+   In `metal3-dev-env` this is done through the scripts in `scripts/v1alphaX/`
+ directory.
 
 5. Wait for the target management cluster to be up and running and once it is up
-  get the kubeconfig for the new target management cluster.
+ get the kubeconfig for the new target management cluster.
 
 6. Use the new cluster’s kubeconfig to install the ironic-components in the
-  target cluster.
+ target cluster.
 
 7. Use clusterctl init with the new cluster’s kubeconfig to install the provider
-  components.
+ components.
```

Comment:
I feel like we don't need to give examples with dev-env, but just generic instructions on how to perform move.

Reply:
done

Reply:
This has to stay like this because otherwise markdownlint has issue with it.